scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -476,4 +532,11 @@ object Config {
     is.close()
     md5Hex(bytes)
   }
+
+  private[this] def buildInj[T: ExternalizerInjection: ExternalizerCodec]: Injection[T, String] =
```

Comment:
while we're at it, why not stack in GzippedBase64String:

https://github.com/twitter/bijection/blob/develop/bijection-core/src/main/scala/com/twitter/bijection/BinaryBijections.scala#L33

Might make the jobconfs slightly smaller.